packages/node/test/unit/node.spec.ts

```diff
@@ -1,12 +1,55 @@
+import * as cf from "@counterfactual/cf.js";
+import * as ethers from "ethers";
+import firebase from "firebase";
+import firebasemock from "firebase-mock";
+
 import Node from "../../src/node";
 
-describe("Node", () => {
+describe("Node operations", () => {
+  const privateKey = process.env.npm_package_config_unlockedAccount0!;
+  const provider = ethers.getDefaultProvider() as ethers.providers.Web3Provider;
+  let firestore: firebase.firestore.Firestore;
+  let networkContext: Map<string, cf.types.Address>;
+
+  beforeEach(() => {
+    firestore = new firebasemock.MockFirestore();
+    networkContext = new Map<string, cf.types.Address>();
+    networkContext.set("AppRegistry", ethers.constants.AddressZero);
+  });
+
   it("exists", () => {
     expect(Node).toBeDefined();
   });
 
   it("can be instantiated", () => {
-    const node = new Node();
+    const node = new Node(privateKey, provider, firestore, networkContext);
     expect(node).toBeDefined();
   });
+
+  it("can create a multisig", async () => {
+    const node = new Node(privateKey, provider, firestore, networkContext);
+    const multisig = await node.createMultisig(ethers.constants.AddressZero);
+    expect(multisig).toBeDefined();
+  });
+
+  it("can accept call to get app instances", done => {
+    const node = new Node(privateKey, provider, firestore, networkContext);
+    const requestId = "1";
+    const req: cf.types.Node.MethodRequest = {
+      requestId,
+      type: cf.types.Node.MethodName.GET_APP_INSTANCES,
+      params: {}
+    };
+
+    // Set up listener for the method response
+    node.on(requestId, (res: cf.types.Node.MethodResponse) => {
```

Comment:
This doesn't look right to me. The event we're listening for shouldn't be the `requestId`, it should probably be `req.type` and the response object would have a `requestId` property.

Reply:
Seconded.

Reply:
That also makes sense.

This is why I brought this up earlier and I thought we agreed the event type would be the request ID, hence this.

Will update.